ReleaseNotes.md

```diff
@@ -1,6 +1,8 @@
 # ALiWa - Alias Light Wallet Release Notes
 
 ## Changelog
+Adding type native currency for sending & receiving
```

Comment:
The changes should be listed below the corresponding release version, not above.